baybe/acquisition/acqfs.py

```diff
@@ -0,0 +1,73 @@
+"""Available acquisition functions."""
+
+from typing import ClassVar
+
+from attrs import define, field
+from attrs.validators import ge
+
+from baybe.acquisition.base import AcquisitionFunction
+
+
+@define(frozen=True)
+class PosteriorMean(AcquisitionFunction):
+    """Posterior Mean."""
+
+    _abbreviation: ClassVar[str] = "PM"
+
+
+@define(frozen=True)
+class qExpectedImprovement(AcquisitionFunction):
+    """Monte Carlo based expected improvement."""
+
+    _abbreviation: ClassVar[str] = "qEI"
+
+
+@define(frozen=True)
+class ExpectedImprovement(AcquisitionFunction):
+    """Analytical expected improvement."""
+
+    _abbreviation: ClassVar[str] = "EI"
+
+
+@define(frozen=True)
+class qProbabilityOfImprovement(AcquisitionFunction):
+    """Monte Carlo based probability of improvement."""
+
+    _abbreviation: ClassVar[str] = "qPI"
+
+
+@define(frozen=True)
+class ProbabilityOfImprovement(AcquisitionFunction):
+    """Analytical probability of improvement."""
+
+    _abbreviation: ClassVar[str] = "PI"
+
+
+@define(frozen=True)
+class qUpperConfidenceBound(AcquisitionFunction):
+    """Monte Carlo based upper confidence bound."""
+
+    _abbreviation: ClassVar[str] = "qUCB"
+
+    beta: float = field(converter=float, validator=ge(0.0), default=0.2)
```

Comment:
thats the botorch default, before it was just randomly chosen or at least I dont remember why we chose that number. I guess adding 1 std seems like a canonical way, but imo its better following the defaults to not cause unexpected behavior or users

 Is this OK?

Reply:
I have nothing against changing (as I said, I mention this only because you asked me the exact same question when I change it in the hotfix :smile:). Only potential issue: we are strictly speaking not backward compatible if someone used the "(q)UCB" string construction method. One COULD quite easily capture this in the deprecation utility as well, raising an according warning. Not the highest priority but maybe better be safe than sorry? What do you think?

Reply:
its somewhat-backwards compatible in the sense that not error will be produced, its a different underlying algo now tho
not sure we need to deprecate that imo lets not do it